src/lambda_function.py

```diff
@@ -50,7 +35,25 @@
 session = requests.Session()
 session.headers = {'Authorization': auth_token}
 
-host = "https://cloudapi-dmk-community-test-lambda-endpoint.n.dwelo.com"    # TODO - parameterize
+# Possible values of STAGE are ['dev', 'qa', 'staging', 'prod']
+STAGE = os.environ.get('STAGE').lower()
+
+S3_CERT_FOLDER = {
+    'dev': 'dev',
+    'qa': 'qa',
+    'staging': 'stg',
+    'prod': 'prd',
+}
+
+DMK_COMMUNITIES_URL = "/v3/integrations/dormakaba/lambda/"
+
+CLOUD_HOST = {
+    # Feature branches require manually adding an env var with the host, i.e. https://cloudapi-BRANCH-SLUG.n.dwelo.com
+    'dev': os.environ.get('FEATURE_BRANCH_HOST', 'https://cloudapi.n.dwelo.com'),
```

Comment:
Would we want to use this against feature branches? I would assume we would probably just want this pointed to either prod or qa just to make sure that our integration is still in a healthy state.